ngraph/core/include/ngraph/node.hpp

```diff
@@ -45,6 +47,7 @@ namespace ngraph
     class AttributeVisitor;
     class Variant;
     class Node;
+    class EvaluationContext;
```

Comment:
Why do you use forward declaration? Can we just include the right header?

Reply:
EvaluationContext uses Variant, Variant uses Node, Node uses EvaluationContext
we need forward declaration here

Reply:
fixed